packages/html-elements/package.json

```diff
@@ -38,15 +38,21 @@
     "projects": [
       {
         "preset": "jest-expo/ios",
-        "modulePathIgnorePatterns": ["<rootDir>/babel/"]
+        "modulePathIgnorePatterns": [
```

Comment:
It seems to be an unrelated formatting change 😅 I would avoid that if the PR has nothing to do with this particular package.
For example it's then recognized as an unpublished change, but in fact it doesn't have to be published.

Reply:
Reverted